Csv/src/Akka.Streams.Csv/CsvToDictionaryStage.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Akka.IO;
+using Akka.Streams.Stage;
+
+namespace Akka.Streams.Csv
+{
+    /// <summary>
+    /// Internal API: Converts incoming ImmutableList&lt;ByteString&gt; to Dictionary&lt;string, ByteString&gt;
```

Comment:
Instead of a naked reference to another class, ```Immutable&lt;ByteString&gt;```, can you use the appropriate xml-doc xref instead, e.g.  ```<see cref="ImmutableList{ByteString}"/>``` and ```<see cref="Dictionary{string, ByteString}"/>```?

Ditto for the rest of the references.

It will help out with the generated documentation. :)